File: Scene.java
package com.example.aroundtheworld.Game;

import android.graphics.Canvas;
import android.view.MotionEvent;

public interface Scene {
    public void update();
    public void draw(Canvas canvas);
    public void terminate();
    public void receiveTouch(MotionEvent event);
    public ObstacleManager getObstacleManager();
}

File: app/src/main/java/com/example/myapplicationn/Scene.kt
package com.example.myapplicationn

import android.graphics.Canvas
import android.view.MotionEvent

interface Scene {
    fun update()
    fun draw(canvas: Canvas?)
    fun terminate()
    fun receiveTouch(event: MotionEvent?)
    fun getObstacleManager(): ObstacleManager?
}